.travis.yml

```diff
@@ -12,6 +12,7 @@ addons:
   apt:
     packages:
       - expect-dev # provides unbuffer utility
+      - "xmlsec1"
```

Comment:
could you get rid of travis here if you have time? :pray: :smile:

Reply:
Can we add apt dependencies on github actions? 👀

Reply:
sure

Reply:
allright, it's done!